mmv1/products/netapp/activeDirectory.yaml

```diff
@@ -0,0 +1,190 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'activeDirectory'
+description: |
+  ActiveDirectory is the public representation of the active directory config.
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'QUICKSTART_TITLE': 'https://cloud.google.com/netapp/volumes/docs/reference/rest/v1/projects.locations.activeDirectories'
+  api: 'https://cloud.google.com/netapp/volumes/docs/reference/rest/v1/projects.locations.activeDirectories'
+
+base_url: projects/{{project}}/locations/{{location}}/activeDirectories
+
+self_link: projects/{{project}}/locations/{{location}}/activeDirectories/{{active_directory_id}}
+
+create_url: projects/{{project}}/locations/{{location}}/activeDirectories?activeDirectoryId={{active_directory_id}}
+
+create_verb: :POST
+
+update_url: projects/{{project}}/locations/{{location}}/activeDirectories/{{active_directory_id}}
+
+update_verb: :PATCH
+
+update_mask: true
+
+delete_url: projects/{{project}}/locations/{{location}}/activeDirectories/{{active_directory_id}}
+
+delete_verb: :DELETE
+
+autogen_async: true
+
+async: !ruby/object:Api::OpAsync
+
+  operation: !ruby/object:Api::OpAsync::Operation
+    base_url: '{{op_id}}'
+
+id_format: 'projects/{{project}}/locations/{{location}}/activeDirectories/{{active_directory_id}}'
+import_format: ['projects/{{project}}/locations/{{location}}/activeDirectories/{{active_directory_id}}']
+
+parameters:
+  - !ruby/object:Api::Type::String
+    name: 'location'
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      LOCATION_DESCRIPTION
+  - !ruby/object:Api::Type::String
+    name: 'active_directory_id'
+    description:
+      'ID of the active directory to create
+      `[_a-zA-Z0-9][-_.a-zA-Z0-9]*`.'
+    required: true
+    immutable: true
+    url_param_only: true
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'Active_Directory_Create'
```

Comment:
update this test name to match the renamed `netapp_active_directory_basic`

Reply:
Updated!